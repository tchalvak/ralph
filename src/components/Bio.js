import React from 'react'
import PropTypes from 'prop-types'

const Bio = props => {
    const person = props.person || {}
    const { children } = props
    return (
        <section>
            <h1>{person.name}</h1>
            <h2>{person.phone}</h2>
            {children}
        </section>
    )
}

Bio.propTypes = {
    person: PropTypes.object.isRequired,
}

export default Bio
