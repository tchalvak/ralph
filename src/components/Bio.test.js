import React from "react"
import renderer from "react-test-renderer"
import Bio from "./Bio"

describe("Bio", () => {
  it("renders correctly", () => {
    const tree = renderer.create(<Bio />).toJSON()
    expect(tree).toMatchSnapshot()
  })
})