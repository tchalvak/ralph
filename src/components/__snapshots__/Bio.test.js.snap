// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Bio renders correctly 1`] = `
<section>
  <h1 />
  <h2 />
</section>
`;
